File: BBIS.Api/ClientApp/src/models/TestOrder/CreateTestOrderDto.ts
import { CreateCrossMatchOrderDto } from './CreateCrossMatchOrderDto';
export class CreateTestOrderDto {
    PatientId: Guid = '';
    ReservationId: Guid = '';
    TestOrders: Array<Guid> = [];
    CrossMatchTestOrders: Array<CreateCrossMatchOrderDto> = [];
}

File: BBIS.Application/DTOs/TestOrder/CreateTestOrderDto.cs
namespace BBIS.Application.DTOs.TestOrder
{
    public class CreateTestOrderDto
    {
        public Guid PatientId { get; set; }
        public Guid ReservationId { get; set; }

        public List<Guid> TestOrders { get; set; }
        public List<CreateCrossMatchOrderDto> CrossMatchTestOrders { get; set; }
    }
        
}
